openmeetings-web/src/main/java/org/apache/openmeetings/web/app/Application_de.properties.xml

```diff
@@ -1004,7 +1004,7 @@ see https://openmeetings.apache.org/LanguageEditor.html for Details
 	<entry key="upload.dlg.choose.title"><![CDATA[Datei auswählen]]></entry>
 	<entry key="upload.dlg.convert.title"><![CDATA[Converting...]]></entry>
 	<entry key="lbl.settings.whiteboard"><![CDATA[Whiteboard Settings]]></entry>
-	<entry key="video.settings.https.required"><![CDATA[Camera/microphone most probably will not work in insecure environment.]]></entry>
+	<entry key="video.settings.https.required"><![CDATA[Camera/microphone most probably will not work in insecure environment. Try using https.]]></entry>
```

Comment:
Maybe you can translate it to German? ;)

Reply:
done

Reply:
Looks better now